### ISSUE TITLE
dials.symmetry says lysozyme is P4

### ISSUE THREAD

**Issue by graeme-winter** [dials/dials#1742]
[symm_wrong.zip](https://github.com/dials/dials/files/6649041/symm_wrong.zip)

That is not a problem per se

```
Estimation of resolution for Laue group analysis

Removing 17 Wilson outliers with E^2 >= 16.0
Resolution estimate from <I>/<σ(I)> > 4.0 : 2.43
Resolution estimate from CC½ > 0.60: 5.65
High resolution limit set to: 2.43
Selecting 3145 reflections with d > 2.43

Input crystal symmetry:
Unit cell: (37.4233, 79.3749, 79.6591, 89.8611, 89.6826, 89.9018)
Space group: P 1 (No. 1)
Change of basis op to minimum cell: a,b,c
Crystal symmetry in minimum cell:
Unit cell: (37.4233, 79.3749, 79.6591, 89.8611, 89.6826, 89.9018)
Space group: P 1 (No. 1)
Lattice point group: P 4/m m m (c,a,b)

Overall CC for 1559 unrelated pairs: 0.097
Estimated expectation value of true correlation coefficient E(CC) = 0.991
Estimated sd(CC) = 1.023 / sqrt(N)
Estimated E(CC) of true correlation coefficient from identity = 0.920

--------------------------------------------------------------------------------

Scoring individual symmetry elements

+--------------+--------+------+-----+----+---------------+
|   likelihood |   Z-CC |   CC |   N |    | Operator      |
|--------------+--------+------+-----+----+---------------|
|        0.423 |   4.23 | 0.62 |  48 |    | 1 |(0, 0, 0)  |
|        0.84  |   8.64 | 0.86 | 356 | ** | 4 |(1, 0, 0)  |
|        0     |   0    | 0    |   0 |    | 2 |(1, 0, 0)  |
|        0.456 |   6.94 | 0.69 | 398 |    | 2 |(0, 1, 0)  |
|        0.552 |   1.17 | 0.6  |   4 | *  | 2 |(0, 0, 1)  |
|        0.348 |   6.36 | 0.64 | 118 |    | 2 |(0, 1, 1)  |
|        0.459 |   6.95 | 0.69 | 296 |    | 2 |(0, -1, 1) |
+--------------+--------+------+-----+----+---------------+

--------------------------------------------------------------------------------

Scoring all possible sub-groups

+-------------------+----+--------------+----------+--------+--------+------+-------+---------+--------------------+
| Patterson group   |    |   Likelihood |   NetZcc |   Zcc+ |   Zcc- |   CC |   CC- |   delta | Reindex operator   |
|-------------------+----+--------------+----------+--------+--------+------+-------+---------+--------------------|
| P 4/m             |    |        0.359 |     0.93 |   6.8  |   5.88 | 0.84 |  0.69 |     0.4 | b,c,a              |
| P 4/m m m         |    |        0.167 |     6.2  |   6.2  |   0    | 0.75 |  0    |     0.4 | b,c,a              |
| P 1 2/m 1         |    |        0.084 |    -4.17 |   3.1  |   7.27 | 0.63 |  0.75 |     0.3 | -a,-c,-b           |
| P m m m           |    |        0.071 |    -2.64 |   4.74 |   7.38 | 0.69 |  0.78 |     0.3 | a,b,c              |
| P 1 2/m 1         |    |        0.068 |    -2.29 |   4.23 |   6.52 | 0.62 |  0.75 |     0.3 | -b,-a,-c           |
| P -1              |    |        0.068 |    -2.29 |   4.23 |   6.52 | 0.62 |  0.75 |     0   | a,b,c              |
| C 1 2/m 1         |    |        0.058 |    -0.66 |   5.75 |   6.41 | 0.69 |  0.76 |     0.3 | b+c,-b+c,a         |
| P 1 2/m 1         |    |        0.057 |    -0.67 |   5.75 |   6.42 | 0.69 |  0.78 |     0.2 | a,b,c              |
| C 1 2/m 1         |    |        0.036 |    -1.17 |   5.4  |   6.56 | 0.66 |  0.76 |     0.4 | b-c,b+c,a          |
| C m m m           |    |        0.031 |    -0.47 |   5.96 |   6.43 | 0.68 |  0.77 |     0.4 | b-c,b+c,a          |
+-------------------+----+--------------+----------+--------+--------+------+-------+---------+--------------------+

Best solution: P 4/m
Unit cell: 
Reindex operator: b,c,a
Laue group probability: 0.359
Laue group confidence: 0.262

+-------------------+--------------------------+
| Patterson group   | Corresponding MX group   |
|-------------------+--------------------------|
| P 4/m             | P 4                      |
+-------------------+--------------------------+
Saving reindexed experiments to symmetrized.expt in space group P 4
Saving 33213 reindexed reflections to symmetrized.refl
```

but if I set `laue_group=P422` because I know better I get 

```
Crystal:
    Unit cell: 62.052, 62.052, 79.659, 90.000, 90.000, 90.000
    Space group: P 4 2 2
    U matrix:  {{ 0.0447,  0.9738,  0.2230},
                { 0.5444,  0.1634, -0.8228},
                {-0.8376,  0.1582, -0.5228}}
    B matrix:  {{ 0.0161,  0.0000,  0.0000},
                {-0.0000,  0.0161,  0.0000},
                {-0.0000,  0.0000,  0.0126}}
    A = UB:    {{ 0.0007,  0.0157,  0.0028},
                { 0.0088,  0.0026, -0.0103},
                {-0.0135,  0.0025, -0.0066}}
    A sampled at 101 scan points
```

output which _can't be good_

**Comment by kmdalton** [comment]
On my favorite lysozyme dataset, `dials.symmetry` gets the point group right but totally whiffs on the screw axes. It assigns `P 4 2 2` as the most likely group. Pointless has no trouble with these data. It recommends the enantiomorphic group, `P 41 21 2`. 

[integrated.expt.gz](https://github.com/dials/dials/files/8150139/integrated.expt.gz)
[integrated.mtz.gz](https://github.com/dials/dials/files/8150140/integrated.mtz.gz)
[integrated.refl.gz](https://github.com/dials/dials/files/8150141/integrated.refl.gz)

Are there some knobs that could help with this? Halp, @graeme-winter

**Comment by kmdalton** [comment]
As far as I can tell, the contents of `symmetrized.refl` should have plenty of signal. 
![image](https://user-images.githubusercontent.com/2790777/155917846-00d76826-c0de-4d7a-833d-5718667486c6.png)
![image](https://user-images.githubusercontent.com/2790777/155917890-cee13f43-267c-4646-88a4-1d67ec3e3cfa.png)

**Comment by jbeilstenedmands** [comment]
Hi @kmdalton 
It looks like there is still a fair amount of signal in the expected absent positions:
```+--------------+---------+---------------+--------------+---------------+--------------+-------------------+------------------+
| Screw axis   |   Score |   No. present |   No. absent |   <I> present |   <I> absent |   <I/sig> present |   <I/sig> absent |
|--------------+---------+---------------+--------------+---------------+--------------+-------------------+------------------|
| 41c          |       0 |            10 |           29 |      1569.4   |      401.086 |            97.103 |           30.685 |
| 21a          |       0 |            21 |           21 |       655.719 |        4.563 |            56.705 |            4.417 |
| 42c          |       0 |            20 |           19 |      1361.99  |        4.508 |            88.712 |            4.561 |
+--------------+---------+---------------+--------------+---------------+--------------+-------------------+------------------+
```
I think your plots also show this? (You can see dials plot in `dials.symmetry.html`)
I'm quite surprised that pointless assigns `P 41 21 2` in this case, given the signal that's there. Is there an issue with high background in your data?

**Comment by jbeilstenedmands** [comment]
Okay, that makes sense. I would certainly be happy for a Fourier analysis to be implemented as an alternative algorithm, and perhaps both the Fourier analysis and current algorithm could be used and reported by default. I think the current algorithm would be better able to identify true weak intensities in the absent positions, but is more likely to wrongly assign cases like your example.

**Comment by kmdalton** [comment]
Is there a plan for implementing Fourier analysis in `dials.symmetry`? I have pretty limited bandwidth for new projects right now, but I think this is on the critical path to circumventing `ccp4i`.

**Comment by jbeilstenedmands** [comment]
Sorry but it's not something I'm going to be able to work on any time soon, there may be interest from other developers though.

**Comment by kmdalton** [comment]
No problem, @jbeilstenedmands. I think much of the basic machinery is covered in my little `numpy` example above. Maybe another developer can run with it. I'll file a new issue that specifically requests this feature and links to our discussion here. Who knows? Maybe I will have time to revisit this later.

**Comment by graeme-winter** [comment]
I'll have a go at sneaking in some time, no promises mind!

**Comment by kmdalton** [comment]
FYI -- running @graeme-winter 's data through Pointless gives the following output:
![image](https://user-images.githubusercontent.com/2790777/182428885-4f39acc6-9c94-4bb7-b3ed-aa7a55685432.png)

There are nowhere near enough axial reflections to properly call screw axes. However, it does nail the Laue group.

**Comment by kmdalton** [comment]
resolved by #2184
